### PR TITLE
Ab test creator name discovery

### PR DESCRIPTION
**Pull request by Scollaco**
# What
A/B test to show (or not) the creator's name on Discovery cell.

# Why
This test will give us a feedback of how the infrastructure is working, so we can add more tests in the future.

### CODE REVIEW

File: Library/ViewModels/ProjectPamphletMainCellViewModel.swift [hunk 1]
```diff
@@ -105,6 +103,9 @@ public protocol ProjectPamphletMainCellViewModelOutputs {
   /// Emits a boolean that determines if the project state label should be hidden.
   var stateLabelHidden: Signal<Bool, NoError> { get }
 
+  /// Emits a string to use for the stats stack view accessibility value.
+  var statsStackViewAccessibilityLabel: Signal<String, NoError> { get }
```
Comment (cdolm92) [inline Library/ViewModels/ProjectPamphletMainCellViewModel.swift]: 👍, thanks for alphabetizing!

File: KsApi/models/Config.swift [hunk 2]
```diff
@@ -2,6 +2,18 @@ import Argo
 import Curry
 import Runes
 
+public enum Experiment {
+
+  public enum Name: String {
+    case iosTest = "ios_test_test"
```
Comment (justinswart) [inline KsApi/models/Config.swift]: Is `ios_test_test` the name being returned from production at this stage?

File: Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift [hunk 3]
```diff
@@ -251,6 +258,7 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
 
   internal func configureWith(value: Project) {
     self.viewModel.inputs.configureWith(project: value)
+    self.viewModel.inputs.current(configuration: AppEnvironment.current.config)
```
Comment (justinswart) [inline Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift]: We don't need this input, `AppEnvironment` is globally accessible so we can just access it where needed in the view model.

File: Library/ViewModels/DiscoveryViewModel.swift [hunk 4]
```diff
@@ -176,6 +180,16 @@ DiscoveryViewModelOutputs {
     self.viewWillAppearProperty.value = animated
   }
 
+  private static func toString(_ dictionary: [String: String]?) -> String {
```
Comment (justinswart) [inline Library/ViewModels/DiscoveryViewModel.swift]: `abExperiments` is optional on `Config` so we don't need to return the default empty string. Also - wouldn't it be better to make `Experiment` conform to `Codable` and then we could just use  `JSONDecoder`'s `KeyedDecodingContainer` to encode/decode this? Maybe I'm missing something.

File: KsApi/models/Config.swift (hunk 2 above)
Comment (Scollaco) [reply KsApi/models/Config.swift]: Yes, but I think it will be removed as soon as we implement real tests.

**Pull request merged**